app/constants/view.ts

```diff
@@ -13,6 +13,7 @@ export const TEAM_SIDEBAR_WIDTH = 72;
 export const TABLET_HEADER_HEIGHT = 44;
 export const TABLET_SIDEBAR_WIDTH = 320;
 
+export const IOS_STATUS_BAR_HEIGHT = 20;
```

Comment:
I am a bit worried about the hardcoded value. There is no better way to get this value?